src/communication/azext_communication/tests/latest/test_communication_chat_scenario.py

```diff
@@ -265,7 +265,7 @@ def test_chat_remove_participants(self, communication_resource_info):
             self.cmd('az communication chat participant remove --thread {thread_id} --user {user_id} --yes')
 
         # assert 'The initiator doesn\'t have the permission to perform the requested operation.' in str(raises.exception)
-        # For now rest endpoint returns a 500 error, so we are checking for that
```

Comment:
It's not necessary to release a new version if only the test code changed.

Reply:
@kairu-ms, reverted the version upgrade changes